modules/sievefilters/site.js

```diff
@@ -186,6 +190,455 @@ var hm_sieve_possible_actions = function() {
     ];
 };
 
+    const add_filter_condition = Hm_Filters.add_filter_condition;
+    const add_filter_action = Hm_Filters.add_filter_action;
+
+
+/**************************************************************************************
+*                                      MODAL EVENTS
+**************************************************************************************/
+var hm_sieve_button_events = function (edit_filter_modal, edit_script_modal) {
+
+  $(document)
+    .off("click")
```

Comment:
You are removing all 'click' event handlers on the document element - this will have severe side-effects. If you want to avoid attaching duplicate event handlers, use proper event filtering when removing event handlers - they need to be the same as the ones you use in the 'on click' handler.